### PR TITLE
Generating headers for updated Execute command

### PR DESCRIPTION
**Pull request by EliotRossRob**
The PR is to bring in ExecuteCommand.1.3.hpp to allow the use of an output field in the response

### CODE REVIEW

File: src/types/reg/udral/physics/acoustics/Note_0_1.hpp [hunk 1]
```diff
@@ -5,9 +5,9 @@
 // Checking this file under version control is not recommended since metadata in this header will change for each
 // build invocation. TODO: add --reproducible option to prevent any volatile metadata from being generated.
 //
-// Generator:     nunavut-2.3.1 (serialization was enabled)
```
Comment (aentinger) [inline src/types/reg/udral/physics/acoustics/Note_0_1.hpp:8]: I'd say you are using quite an outdated nnvg version. Have you used the script supplied with this library?

**Review by aentinger — CHANGES_REQUESTED** [review]
Please use build scripts shipped with this library.

File: src/types/reg/udral/physics/acoustics/Note_0_1.hpp (hunk 1 above)
Comment (EliotRossRob) [reply src/types/reg/udral/physics/acoustics/Note_0_1.hpp:8]: I used the update_cpp_header.sh file in the extras folder to generate

Comment (aentinger) [reply src/types/reg/udral/physics/acoustics/Note_0_1.hpp:8]: How about you follow the instructions outlined here: https://github.com/107-systems/107-Arduino-Cyphal/blob/main/extras/script/README.md 🙏

Comment (EliotRossRob) [reply src/types/reg/udral/physics/acoustics/Note_0_1.hpp:8]: my apologies

**Review by aentinger — APPROVED** [review]

**Pull request merged**